app/views/components/view_application.scala.html

```diff
@@ -114,41 +114,52 @@ <h3>@messages("view.application.your.record.text")</h3>
                 <div class="grid-row">
                     <div class="column-two-thirds">
                         <h3>@messages("confirmation.heading2.whatNext")</h3>
-                        <p class="govuk-body">@messages("view.application.paragraph.whatNext")</p>
+                        @if(!pdf.sendingSample) {
+                            <p class="govuk-body">@messages("view.application.paragraph.whatNext")</p>
+                        }
 
                         @if(pdf.sendingSample) {
```

Comment:
```suggestion
                        @if(!pdf.sendingSample) {
                            <p class="govuk-body">@messages("view.application.paragraph.whatNext")</p>
                        } else {
```
This could be an if-else